doc/libpmemkv.7.md

```diff
@@ -87,14 +87,16 @@ TBB and libpmemobj-cpp packages are required.
 
 This engine requires the following config parameters (see **libpmemkv_config**(3) for details how to set them):
 
-* **path** -- Path to the database file
+* **path** -- Path to the database file. Can only be specified when oid is not specified.
 	+ type: string
-* **force_create** -- If 0, pmemkv opens file specified by 'path', otherwise it creates it
+* **force_create** -- If 0, pmemkv opens file specified by 'path', otherwise it creates it. Can only be specified when path is specified.
 	+ type: uint64_t
 	+ default value: 0
-* **size** --  Only needed when force_create is not 0, specifies size of the database [in bytes]
+* **size** --  Only needed when force_create is not 0, specifies size of the database [in bytes]. Can only be specified when path is specified.
 	+ type: uint64_t
 	+ min value: 8388608 (8MB)
+* **oid** -- Pointer to oid which points to engine data. If oid is null, engine will allocate new data, otherwise it will use existing data. Can only be specified when path is not specified.
```

Comment:
For C++ it's a persistent pointer, and oid is just an implementation detail. 

Anyway, this should talk about libpmemobj, and that it requires that a PMEMobjpool is open when pmemkv is running. This might be obvious to us, but...

Reply:
Here, user does not know the type of pointer (it is implementation detail of engine) and for persistent_ptr it is required to specify the type. We do not have un-typed persistent_ptr. Because of that, we can only use PMEMoid.

Reply:
`persistent_ptr<void>`? Does that work?
Anyway, my point is that this talks about an "oid" without any context in the man page itself. At the very least, there should be a reference to libpmemobj's manpage.